### PR TITLE
Only run CreateDailySpoolFiles on weekdays

### PR DESCRIPTION
**Pull request by markolson**
Per Shay, we should only send Spool Files on weekdays that do not have observed federal holidays on them

### CODE REVIEW

**Comment by markolson** [comment]
Should I move the "weekday" logic into the job itself?

File: config/sidekiq_scheduler.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # at the specified interval.
 
 CreateDailySpoolFiles:
-  cron: "0 3 * * * America/New_York"
+  cron: "0 3 * * 1-5 America/New_York"
```
Comment (akainic) [inline config/sidekiq_scheduler.yml]: What is this for?

Comment (markolson) [inline config/sidekiq_scheduler.yml]: This change sets the job to only run on "days of the week 1-5", where 0 is Sunday, and 1 is Monday. Cron is fun.

https://crontab.guru/#0_3_*_*_1-5

**Comment by markolson** [comment]
@akainic the division right now its that `sidekiq_scheduler` is set to only attempt to run the job on weekdays (`MON-FRI` now), and the `federal_holiday?` check in the job itself will exit right away if a holiday is observed on the current day.

**Pull request merged**